hasher-matcher-actioner/CONTRIBUTING.md

```diff
@@ -86,3 +93,69 @@ Until a Syncer module is created, test threatexchange data needs to be manually
 3. Upload some target and no target images.
 
 This should run the images through the existing PDQ hasher and matcher lambda functions.
+
+
+## Running Lambdas locally
```

Comment:
worth noting that both this and the run api commands not only require tf outputs but also need local aws admin auth to work.